teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(specialization, age, parent_permission: true, name: 'Unknown')
+    super(age: age, parent_permission: parent_permission, name: name)
+    @specialization = specialization
+  end
```

Comment:
Well! great job on updating the parameter for the super method. but still need some updates.
Currently, when I try to create a new Teacher, I get an error of `/person.rb:5:in `initialize': wrong number of arguments`.  I will recommend you follow the previous reviewer's instruction and follow it fully and update the parameter further. 🙏 
```suggestion
  def initialize(specialization, age, parent_permission: true, name: 'Unknown')
    super(age, parent_permission: parent_permission, name: name)
    @specialization = specialization
  end
```